cassdegrees/templates/bulkupload.html

```diff
@@ -25,7 +26,7 @@
 
     {% for type in upload_type %}
         <div id="{{ type }}Page" hidden>
-            <form class="anuform" action="/bulk_upload/?type={{ type }}" method="post" enctype="multipart/form-data">
+            <form class="anuform" action="/admin/bulk_upload/?type={{ type }}" method="post" enctype="multipart/form-data">
```

Comment:
Can this be relative? (i.e `action="type={{ type }}"`)?

Reply:
Yep, tried that and it works!